### ISSUE TITLE
Possible additional tarball exclusions to reduce size

### ISSUE THREAD

**Issue by iskunk** [chromium-linux-tarballs/chromium-tarballs#7]
I compared the upstream tarball with the one generated here to see what else could be cut out as nonessential.

I see `third_party/rust-src/` and `third_party/llvm/` are gone; the following seem to be other big ones still present:
```
third_party/instrumented_libs/  (about 2 GB)
build/linux/debian_bullseye_{amd64,i386}-sysroot/  (over 500 MB each)
```
It seems like anything further is going to be small potatoes, like `testing/libfuzzer/fuzzers/wasm_corpus/` (256 MB). Are the above likely to be needed by anyone consuming these tarballs?

**Comment by Kangie** [comment]
This is worth discussing; I'm not explicitly trying to make smaller tarballs but there is a case to be made for making 'distro' tarballs, and we're already a few steps down this path. Also since I'm hosting the S3 bucket the smaller tarballs make a difference to me!

The Sysroot is documented here: https://chromium.googlesource.com/chromium/src/+/master/docs/linux/sysroot.md

TL;DR - upstream builds against a very old debian rather than system libs by default. Distros can probably safely ignore this, and there's a script to rebuild the sysroot if it's ever required.

The instrumented libs are documented here: https://www.chromium.org/developers/testing/instrumented-libraries-for-dynamic-tools/

Looks like we'd be fine without both of these, at a glance.

It may even be worth upstreaming these as 'nonessential files'; the `-lite` tarball is supposed to be much smaller. Tarballs (and their associated infra) don't see much love upstream.

**Comment by iskunk** [comment]
Yeah, my impression is that users coming from the context of a particular distro will want to use libraries from that distro, rather than a third-party build (let alone an ancient one). I'll add those dirs to `nonessential_dirs` in `export_tarball.py` over in #3.

I wasn't aware of the existence of a `-lite` tarball, but it appears to be all of 2% smaller than the normal one :-]  Is the difference down to the `nonessential_dirs` list?
